src/ethereum/dao_fork/__init__.py

```diff
@@ -6,3 +6,4 @@
 """
 
 MAINNET_FORK_BLOCK = 1920000
+PREV_FORK = "homestead"
```

Comment:
Would something like:

```python
from ethereum import homestead

PREVIOUS_FORK = homestead
```

work?

Reply:
No, we have to import `ethereum.FORK.trie` (as opposed to just `ethereum.FORK`) somewhere.